deps/Chakra.ICU/Chakra.ICU.Build.props

```diff
@@ -20,6 +20,11 @@
         %(PreprocessorDefinitions)
       </PreprocessorDefinitions>
 
+      <PreprocessorDefinitions>
+        U_DISABLE_RENAMING=1;
+        %(PreprocessorDefinitions)
```

Comment:
If this is only for Win8+, then you likely want `U_PLATFORM_HAS_WINUWP_API=1` as well.

Reply:
These are used back to Windows 7, so that probably wont work. U_DISABLE_RENAMING seems to be supported on its own, right?

Reply:
Ah, if this must go down-level to Win7 then you can't set `U_PLATFORM_HAS_WINUWP_API`. 

Right, the `U_DISABLE_RENAMING` macro should be platform independent in that regard.

Reply:
I'm a bit curious about the Win7 requirement though -- I thought this was just for working around the Server 2012 build environment limitations?

Reply:
The requirement is because builds of ChakraCore are supported down to Windows 7 and VS 2015, so all changes have to be made in that context.

Reply:
Right, ChakraCore != Edge. :)